spellcheck.toml

```diff
@@ -0,0 +1,12 @@
+dev_comments = false
```

Comment:
I would like to see inline comments documenting the justification for these configuration customizations.